packages/webapp/src/containers/SensorReadingsLineChart/index.jsx

```diff
@@ -32,27 +32,13 @@ const SensorReadingsLineChart = ({ readingType, noDataFoundMessage, data }) => {
   );
   let isActive = readingTypeDataExists ? true : false;
   let unit;
-  let subTitle;
-  let weatherStationName;
-  // Reading type differences --
+
   if (readingType === TEMPERATURE) {
-    const weatherStationDataExists = data?.sensorReadingData?.find(
-      (rd) => rd[data.stationName] != '(no data)',
-    );
-    unit = getUnitOptionMap()[ambientTemperature[unitSystem].defaultUnit].label;
-    isActive = weatherStationDataExists || isActive;
```

Comment:
_Definitely_ not intentional, thank you!! 🙏 

I was thinking this was the forecast data rather than the ambient temperature line (and wondering why it had anything to do with the chart being active!) 😬

Reply:
I was thinking the same thing at first!
Changes look good, thank you!